src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -375,10 +388,10 @@ val qualityFormat: Option[FastqQualityFormat] = None,
   private def toWriter(sampleInfo: SampleInfo): DemuxWriter = {
     val sample = sampleInfo.sample
     val isUnmatched = sample.sampleName == UnmatchedSampleId
```

Comment:
Not sure where to put this, but I think the unmatched BAM is called `Undetermined_sequence.fastq.gz?